android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -227,6 +228,67 @@ public void startTrackingCustom(ReadableMap optionsMap) {
         }
     }
 
+    @ReactMethod
+    public void mockTracking(ReadableMap optionsMap, final Promise promise) {
+        if (promise == null) {
+            return;
+        }
+
+        ReadableMap originMap = optionsMap.getMap("origin");
+        double originLatitude = originMap.getDouble("latitude");
+        double originLongitude = originMap.getDouble("longitude");
+        Location origin = new Location("RNRadarModule");
+        origin.setLatitude(originLatitude);
+        origin.setLongitude(originLongitude);
+        ReadableMap destinationMap = optionsMap.getMap("destination");
+        double destinationLatitude = destinationMap.getDouble("latitude");
+        double destinationLongitude = destinationMap.getDouble("longitude");
```

Comment:
would be nice to have utils for these at some point like the collection additions on iOS, not necessary now